### DISCUSSION TITLE
Specifying a Node environment in the new configuration system

### DISCUSSION THREAD

**Discussion by mgnsm** [eslint/eslint#17703] [Q&A] [answered]
What is the equivalent of specifying a Node environment like this in the [new configuration system](https://eslint.org/docs/latest/use/configure/configuration-files-new)?:

```
"env": {
  "node": true
}
```

I could not find any information about this in the docs and the following doesn't seem to work:

```
...
languageOptions: {
  globals: {
    ...globals.node
  }
}
```

`...globals.browser` works though. 

How do I add the Node.js global variables and Node.js scoping to avoid linting errors such as "'module' is not defined" and "'require' is not defined"?

**Comment by mdjermanovic** [comment, marked as answer]
> How do I add the Node.js global variables and Node.js scoping to avoid linting errors such as "'module' is not defined" and "'require' is not defined"?

Can you try with this configuration:

```js
// eslint.config.js

const globals = require("globals");

module.exports = [{
    languageOptions: {
        sourceType: "commonjs",
        globals: {
          ...globals.node
        }
    }
}];
```

**Reply by Rec0iL99** [reply]
@mgnsm can you provide a reproducible example for us to try out?

**Reply by mgnsm** [reply]
`...globals.node` seems to work in version 8.53.0 after all, even without setting the `sourceType` to `commonJs`.
